packages/fxa-admin-panel/src/components/PageRelyingParties/index.tsx

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { ApolloError, gql, useQuery } from '@apollo/client';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { RelyingParty } from 'fxa-admin-server/src/graphql';
+import { DATE_FORMAT } from '../AccountSearch/Account';
+import dateFormat from 'dateformat';
+
+const RELYING_PARTIES_SCHEMA = `
+  relyingParties {
+    id
+    name
+    imageUri
+    redirectUri
+    canGrant
+    publicClient
+    createdAt
+    trusted
+    allowedScopes
+  }
+`;
+
+interface RelyingPartiesData {
+  relyingParties: RelyingParty[];
+}
+
+export const GET_RELYING_PARTIES = gql`
+  query getRelyingParties {
+    ${RELYING_PARTIES_SCHEMA}
+  }
+`;
+
+const Result = ({
+  loading,
+  error,
+  data,
+}: {
+  loading?: boolean;
+  error?: ApolloError;
+  data?: RelyingPartiesData;
+}) => {
+  if (loading) {
+    return <p className="mt-2">Loading...</p>;
+  }
+  if (error) {
+    return <p className="mt-2">An error occurred.</p>;
+  }
+  if (data && data.relyingParties.length > 0) {
+    return (
+      <>
+        {data.relyingParties.map(
+          ({
+            id,
+            name,
+            imageUri,
+            redirectUri,
+            canGrant,
+            publicClient,
+            createdAt,
+            trusted,
+            allowedScopes,
+          }) => (
+            <div key={id}>
+              <h3 className="account-header">{name}</h3>
+              <table className="account-border-info">
+                <tbody>
+                  <tr>
+                    <th>ID</th>
+                    <td>{id}</td>
+                  </tr>
+                  <tr>
+                    <th>Created At</th>
+                    <td>{dateFormat(new Date(createdAt), DATE_FORMAT)}</td>
+                  </tr>
+                  <tr>
+                    <th>Redirect URI</th>
+                    <td>{redirectUri}</td>
+                  </tr>
+                  <tr>
+                    <th>Allowed Scopes</th>
+                    <td>
+                      {allowedScopes ? (
+                        allowedScopes
```

Comment:
Just a nitpick. When I was testing this locally, I noticed that scopes are delimitted by a whitespace, which is kind of hard to read. Mabye we can split on whitespace, and display them as a list?